backend/src/schema/resolvers/reports.spec.js

```diff
@@ -66,70 +73,98 @@ describe('report', () => {
 
       describe('valid resource id', () => {
         beforeEach(async () => {
-          variables = { id: 'u2' }
-        })
-
-        it('creates a report', async () => {
-          await expect(action()).resolves.toEqual({
-            report: { description: 'Violates code of conduct' },
-          })
+          variables = {
+            id: 'u2',
+          }
         })
-
+        /*
+                it('creates a report', async () => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:49:12Z" title="Friday, June 21st 2019, 2:49:12 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -66,70 +73,98 @@ describe('report', () => {
 
       describe('valid resource id', () => {
         beforeEach(async () => {
-          variables = { id: 'u2' }
-        })
-
-        it('creates a report', async () => {
-          await expect(action()).resolves.toEqual({
-            report: { description: 'Violates code of conduct' },
-          })
+          variables = {
+            id: 'u2',
+          }
         })
-
+        /*
+                it('creates a report', async () => {
```


@ogerly @ulfgebhardt never check in disabled (commented) code. You can do this locally for testing. But make sure you either remove the test case (and have a reason for it) or change it. Just don't check in commented code.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-21T13:49:07Z" title="Friday, June 21st 2019, 3:49:07 pm +02:00">Jun 21, 2019</time>_

---


Sorry I missed that. I'll check it out better in the future.